### DISCUSSION TITLE
Welcome to earthengine-py-notebooks Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/earthengine-py-notebooks#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aditya-mishra01** [comment]
I'm not able to see the maps on the notebook viewer 
it only displays this :
![Screenshot (47)](https://github.com/giswqs/earthengine-py-notebooks/assets/55494834/75e3633c-8c00-472b-a1d9-82b745fe1975)

**Comment by Sonet001** [comment]
can anyone tell me how to get the satellite image name from GEE to read in python?
